lib/classes/Variables.js

```diff
@@ -40,7 +40,7 @@ class Variables {
 
     _.deepMapValues(this.service, (property, propertyPath) => {
       if (typeof property === 'string') {
-        const populateSingleProperty = new Promise((resolve) => this
+        const populateSingleProperty = new BbPromise((resolve) => this
```

Comment:
Awesome! Thanks for the suggestion.

The current code in `master` causes a regression in the resolving step for the custom variable syntax. Having this (🔝 ) in place should prevent this and makes this more robust (it was uncovered "by accident")! 👍